.travis.yml

```diff
@@ -11,4 +11,4 @@ before_install:
 script:
   # Load packages and install the required packages
   # Install retriever from source and run tests
-  - docker-compose  run rdata /bin/sh -c 'Rscript load_and_test.R'
+  - docker-compose run -e NOT_CRAN=true rdata /bin/sh -c 'Rscript load_and_test.R'
```

Comment:
I did miss this part. I guess we could probably set the environment, `-e NOT_CRAN=true`, in the docker compose file.
https://github.com/ropensci/rdataretriever/blob/master/docker-compose.yml#L25
But I am happy to go with the command line version

Reply:
Thanks @henrykironde. That's definitely better. I've updated the PR to take this approach instead.